src/interpreter/ByteCodeGenerator.cpp

```diff
@@ -576,6 +576,10 @@ ByteCodeBlock* ByteCodeGenerator::generateByteCode(Context* c, InterpretedCodeBl
                 BlockOperation* cd = (BlockOperation*)currentCode;
                 break;
             }
+            case EnsureArgumentsObjectOpcode: {
+                EnsureArgumentsObject* cd = (EnsureArgumentsObject*)currentCode;
+                break;
```

Comment:
This case statement seems unnecessary because `EnsureArgumentsObjectOpcode` has no allocated register value. Looks same for `BlockOperationOpcode` case which is just above of this case.

Reply:
yes. this is not necessary. just for showing every opcode may needs this stuff. if dont you like it I can remove it

Reply:
I removed it